hive/src/test/java/org/apache/iceberg/hive/TestHiveMetastore.java

```diff
@@ -102,7 +102,7 @@ private TServer newThriftServer(TServerSocket socket, HiveConf conf) throws Exce
         .transportFactory(new TTransportFactory())
         .protocolFactory(new TBinaryProtocol.Factory())
         .minWorkerThreads(3)
-        .maxWorkerThreads(5);
+        .maxWorkerThreads(8);
```

Comment:
Why was this change required?

Reply:
This is used for the unit test. The spark job needs at least three thrift server threads since we have three partitions in the unit test, the maximum value of 5 should be enough while the previous unit tests still not release yet (default release time is 1800s) which cause failed to connect to thrift server.

Reply:
What have previous tests not released? What is the "default release time"?

Reply:
It is a new unit test I added, not related to the previous.

The default release time is referred to `hive.server2.thrift.http.worker.keepalive.time` which is the `stopTimeoutVal` in TThreadPoolServer and is set default 60s ( I referred to `hive.server2.thrift.http.max.idle.time` by mistake..).